src/diepvries/driving_key_field.py

```diff
@@ -12,12 +14,20 @@ class DrivingKeyField:
     defined by a subset of hashkeys.
     """
 
-    #: Name of parent table in the database (always a Link).
-    parent_table_name: str
-    #: Column name in the database.
-    name: str
-    #: Name of the satellite where this field represents a driving key.
-    satellite_name: str
+    def __init__(self, parent_table_name: str, name: str, satellite_name: str):
```

Comment:
Given this is a dataclass, the best way to perform post-init operations is to use `__post_init__`: https://docs.python.org/3/library/dataclasses.html#post-init-processing

Reply:
It's perfect to call `_validate` from `__post_init__` :)